modules/apps/27-interchain-accounts/controller/keeper/migrations.go

```diff
@@ -38,8 +40,8 @@ func (m Migrator) AssertChannelCapabilityMigrations(ctx sdk.Context) error {
 
 			isAuthenticated := m.keeper.scopedKeeper.AuthenticateCapability(ctx, capability, name)
 			if !isAuthenticated {
-				logger.Error(fmt.Sprintf("expected capability owner: %s", controllertypes.SubModuleName))
-				return errorsmod.Wrapf(capabilitytypes.ErrCapabilityNotOwned, "expected capability owner: %s", controllertypes.SubModuleName)
+				logger.Error(fmt.Sprintf("expected capability owner: %s", types.SubModuleName))
+				return errorsmod.Wrapf(capabilitytypes.ErrCapabilityNotOwned, "expected capability owner: %s", types.SubModuleName)
```

Comment:
nit: might be useful to keep `controllertypes` alias

Reply:
generally yes, but iirc this came up a few times when we implemented ICA and due to there being the two submodules it can be useful to be explicit to differentiate. I don't feel particularly strongly about it tho